Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionMatchPatternCocoa.mm

```diff
@@ -45,25 +45,23 @@
 static NSString * const allHostsAndSchemesPattern = @"*://*/*";
 static NSString * const patternFormat = @"%@://%@%@";
 
-NSSet *WebExtensionMatchPattern::validSchemes()
+const WebExtensionMatchPattern::URLSchemeSet& WebExtensionMatchPattern::validSchemes()
 {
-    // FIXME: Add extension schemes to this set.
-    static NSSet<NSString *> *schemes = [NSSet setWithObjects:@"*", @"http", @"https", @"file", @"ftp", nil];
+    // FIXME: Add custom extension schemes to this set somehow.
```

Comment:
Super nit: Not sure the `somehow` adds too much here.

Reply:
It is kind of a hard problem to figure out still. 😅